modules/about-clair.adoc

```diff
@@ -30,6 +30,16 @@ New versions of Clair are regularly released. The source code needed to build Cl
 
 Release artifacts also include the `clairctl` command line interface tool, which obtains updater data from the internet by using an open host.
 
+[discrete]
+[id="clair-releases-474"]
+=== Clair 4.8
+
+Clair 4.8 was released on 24-10-28. The following changes have been made:
+
+* Clair on {productname} now requires that you update the Clair PostgreSQL database from version 4.13 to version 4.15. For more information about this procedure, see link:https://docs.redhat.com/en/documentation/red_hat_quay/3.12/html-single/vulnerability_reporting_with_clair_on_red_hat_quay/index#upgrading-clair-postgresql-database[Upgrading the Clair PostgreSQL database].
```

Comment:
`"PostgreSQL database from version 4.13 to version 4.15"` -->  
"PostgreSQL database from PostgreSQL 13 to PostgreSQL 15" 
or 
"PostgreSQL database from version 13 to version 15"